drupal/templates/drupal-ingress.yaml

```diff
@@ -15,6 +15,9 @@ spec:
   {{- if .Values.ssl.enabled }}
   tls:
   - secretName: {{ .Release.Name }}-tls
+  {{- range $index, $prefix := .Values.domainPrefixes }}
+  - secretName: {{ $.Release.Name }}-tls-{{ $index }}
```

Comment:
This might clash with `.Values.exposeDomains` iterator (https://github.com/wunderio/charts/blob/a8b10c5b894d45f0c85329f96257d82f7d12f4d7/drupal/templates/drupal-ingress.yaml#L61). We need a different prefix for the `domainPrefixes` certificate secret name.

Reply:
It's a clever way, though, defining multiple secrets and domains in a single Ingress. Maybe we should add `exposeDomains` here as well.